xml/System.Resources.Extensions/PreserializedResourceWriter.xml

```diff
@@ -87,12 +90,22 @@
         <Parameter Name="closeAfterWrite" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="typeName">To be added.</param>
-        <param name="closeAfterWrite">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The resource name.</param>
+        <param name="value">The value of the resource in <see cref="T:System.IO.Stream" /> form understood by the type's constructor.</param>
+        <param name="typeName">The assembly qualified type name of the resource.</param>
+        <param name="closeAfterWrite">An optional value that indicates whether, after resources have been written, the stream should be closed (<see langword="true" />) or left open (<see langword="false" />, the default value).</param>
+        <summary>Adds a resource of the specified type represented by a <see cref="T:System.IO.Stream" /> value that is passed to the type's constructor when reading the resource.</summary>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentNullException"><see paramref="name" /> is <see langword="null" />.
```

Comment:
You could also combine the parameters in one sentence:

```suggestion
        <exception cref="T:System.ArgumentNullException"><see paramref="name" />, <see paramref="typeName" />, or <see paramref="value" /> is <see langword="null" /></exception>.
```

Reply:
This would be the first time I see this. I guess it makes sense since the root cause is the same. The -or- seems to be used when the root cause is completely different and unrelated.